cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -399,11 +404,12 @@ def test_change_network_security_groups_per_page_items():
             2.Open Azure Network Manager
             3.Select Network Security Groups
             4.Change items per page
-
-    Bugzilla:
-        1524443
     """
-    pass
+    view = navigate_to(provider, "NetworkSecurityGroup")
+    view.toolbar.view_selector.select('List View')
+    for item_count in [10, 20, 50]:
+        view.paginator.set_items_per_page(item_count)
+        assert view.paginator.items_per_page == len(view.entities.get_all())
```

Comment:
Are you sure this will be true even when there are less number of elements as compared to the items set per page?

Reply:
yeah. I agree that's why I did not opt for large numbers like 1000. 
But I am not sure how can we get to know entities will be less in exist.

Reply:
You could do `assert len(view.entities.get_all()) <= view.paginator.items_per_page`.